src/utils/embedded/embedded.provider.tsx

```diff
@@ -735,32 +735,56 @@ export function EmbeddedProvider({ children }: EmbeddedProviderProps) {
     async (authProviderType: AuthProviderType) => {
       if (user) {
         await supabase.auth.refreshSession();
-
         return;
       }
 
-      // TODO: Handle errors and authentication redirects here:
       try {
-        // setIsLoading(true);
-
         const { url } = await AuthenticationService.authenticate(
           authProviderType
         );
 
         if (url) {
-          // Redirect to Google's OAuth page
-          // Opening the URL on the current tab won't work when Embedded is loaded inside the iframe:
+          // Calculate center position for the popup
+          const width = 500;
+          const height = 600;
+          const left = window.screenX + (window.outerWidth - width) / 2;
+          const top = window.screenY + (window.outerHeight - height) / 2;
+
+          const popup = window.open(
```

Comment:
On what environment(s) did you try this? Just asking because there are some issues that will appear only when running the deployed app inside an iframe hosted on a different domain (like not being able to open the provider URLs within the iframe itself).

Otherwise, I'll be testing this once this PR is merged on the big one.

Might be worth reading: https://blog.huli.tw/2022/04/07/en/iframe-and-window-open/